app/views/editor/rich.html.erb

```diff
@@ -204,11 +204,19 @@
 
       <div class="ple-module-guide col-lg-3">
         <h2>4</h2>
+        
       </div>
-
-      <div class="ple-module-content col-lg-9">
-        <textarea id="text-input" class="ple-textarea form-control"><% if @node && @node.latest && @node.latest.body_rich %><%= @node.latest.body %><% else %><%= params[:body] %><% end %></textarea>
-      </div>
+      <% if @graphobject=="" %>
+        <div class="ple-module-content col-lg-9">
+          <textarea id="text-input" class="ple-textarea form-control"><% if @node && @node.latest && @node.latest.body_rich %><%= @node.latest.body %><% else %><%= params[:body] %><% end %></textarea>
+        </div>
+      <% else %>
+        <div class="ple-module-content col-lg-9">
+          <% x = "Power Tag: simple-data-grapher: i/" + @graphobject[0] + "/" + @graphobject[1] %>
```

Comment:
Hm, can you walk me through exactly what's happening here, or link to the previous PR where it's explained? Thanks!